core/src/cluster_info_vote_listener.rs

```diff
@@ -44,9 +47,12 @@ impl ClusterInfoVoteListener {
             last_ts = new_ts;
             inc_new_counter_info!("cluster_info_vote_listener-recv_count", votes.len());
             let msgs = packet::to_packets(&votes);
-            for m in msgs {
-                sender.send(m)?;
-            }
+            let r = if sigverify_disabled {
+                sigverify::ed25519_verify_disabled(&msgs)
+            } else {
```

Comment:
is the verify_batch() function equivalent to this?

Reply:
verify_batch will use CUDA if its there. This code uses cpu.